tools/phenomenal/nmr/batman.xml

```diff
@@ -3,67 +3,32 @@
      <requirements>
        <container type="docker">docker-registry.phenomenal-h2020.eu/phnmnl/batman</container>
      </requirements>
-     <command><![CDATA[
-runBATMAN.R -i ${nmr_spectra_input}
-            -o \$PWD
-#if str( $options_file ) != None:
-            -p ${options_file}
-#end if
-#if str( $metabolite_template ) != None:
-            -u ${metabolite_template}
-#end if
-#if str( $target_metabolites ) != None:
-            -l ${target_metabolites}
-#end if
+     <command><![CDATA[     
+     runBATMAN.R -i ${nmr_spectra_input} -o \$PWD -p ${options_file} -u ${metabolite_template} -l ${target_metabolites}
        ]]>
      </command>
      <inputs>
-         <param name="nmr_spectra_input" type="data" format="txt" label="NMR Dataset ; A tabulated txt file including p.p.m and spectra"/>
-         <param name="options_file" type="data" format="txt" label="BATMAN options ; A txt file including BATMAN running options, such as number of spectra which must within the range given by the NMR data set. The file name MUST be batmanOptions.txt"/>
-         <param name="metabolite_template" type="data" format="txt" label="pre-defined metabolites template, multi_data_user.csv"/>
-         <param name="target_metabolites" type="data" format="txt" label="target metabolites list"/>
+         <param name="nmr_spectra_input" type="data" format="txt" label="NMR Dataset; A tabulated text file including p.p.m and spectra"/>
+         <param name="options_file" type="data" format="txt" label="BATMAN options; A text file including BATMAN running options, such as number of spectra which must within the range given by the NMR data set. The file name MUST be batmanOptions.txt"/>
+         <param name="metabolite_template" type="data" label="pre-defined metabolites template, multi_data_user.csv"/>
+         <param name="target_metabolites" type="data" label="target metabolites list"/>
      </inputs>
      <outputs>
-         <data name="RelCon" format="tabular" from_work_dir="results/RelCon.txt" label="Relative concentrations"/>
-         <data name="MultipletsPpmShifts" format="tabular" from_work_dir="results/MultipletsPpmShifts.txt" label="A table (M x n) containing the posterior means of multiplets ppm shift for M multiplets as its rows."/>
-         <data name="RelConCreInt" format="tabular" from_work_dir="results/RelConCreInt.txt" label="Relative concentrations in the Confidence Interval of 95%"/>
-
+         <data name="zip_output" format="zip" from_work_dir="results/zip_output.zip" label="BATMAN output files zipped"/>
          <collection name="spectra" type="list" labels="Stack from ${on_string}">
```

Comment:
As discussed on slack, we should keep a few more output files, like RelCon.txt and MultipletsPpmShifts.txt as suggested by Jianliang.